rest-rest/src/Components/RestStep.tsx

```diff
@@ -0,0 +1,396 @@
+import {
+  Form,
+  FormGroup,
+  TextInput,
+  ActionGroup,
+  Button,
+  FileUpload,
+  InputGroup,
+  Checkbox,
+  FormSelect,
+  FormSelectOption,
+} from '@patternfly/react-core';
+import { OpenAPI, OpenAPIV3, OpenAPIV2, OpenAPIV3_1 } from 'openapi-types';
+import { FormEvent, Key, useEffect, useState } from 'react';
+import SwaggerParser from '@apidevtools/swagger-parser';
+import { IStepProps } from '../../../try-catch-eip/kaoto/types/dts/src/types.js';
```

Comment:
I should probably takes this from somewhere else. Do we have the common packages ready in npm?